libs/designsystem/src/lib/components/list/list.component.spec.ts

```diff
@@ -142,16 +143,26 @@ describe('ListComponent', () => {
   describe('function: ngOnInit', () => {
     it('should enable load more, if there is a subscriber to the loadMore event emitter', () => {
       component.loadOnDemand.subscribe((_: LoadOnDemandEvent) => {});
-
       component.ngOnInit();
-
-      expect(component.isLoadOnDemandEnabled).toBeTruthy();
+      expect(component.isLoadOnDemandEnabled).toBeTrue();
     });
 
     it('should disable load more, if there is no subscriber to the loadMore event emitter', () => {
       runNgOnChanges();
 
-      expect(component.isLoadOnDemandEnabled).toBeFalsy();
+      expect(component.isLoadOnDemandEnabled).toBeFalse();
+    });
+  });
+
+  describe('function: enableLoadOnDemand', () => {
+    it('should enable load more, if called with true', () => {
+      component.enableLoadOnDemand(true);
+      expect(component.isLoadOnDemandEnabled).toBeTrue();
+    });
+
+    it('should disable load more, if called with false', () => {
+      component.enableLoadOnDemand(false);
+      expect(component.isLoadOnDemandEnabled).toBeFalse();
```

Comment:
Add a line break to denote a arrange-act-assert pattern (no arrange in this one):
```suggestion
      component.enableLoadOnDemand(false);
      
      expect(component.isLoadOnDemandEnabled).toBeFalse();
```

Reply:
Should be resolved by f1d733e.